inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -181,13 +187,15 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
         if (CPUIter != metaDevices.end()) {
             _loadContext[CPU].isEnabled = true;
             _loadContext[CPU].deviceInfo = *CPUIter;
+            _loadContext[CPU].deviceInfo.config[CONFIG_KEY(PERFORMANCE_HINT)] =
```

Comment:
if the AUTO already configured for LATENCY by the user, then CPU==CPU_HELP again (no need to load the CPU twice)

Reply:
fix it